### DISCUSSION TITLE
Splitmap with planet imaginary

### DISCUSSION THREAD

**Discussion by Niko-La** [opengeos/leafmap#52] [Q&A] [answered]
```m = leafmap.Map()```
Use the above to get multi point pin with pop up attributes.

1. How do you get split map to work and not lose the data.
``` leafmap.split_map(left_layer="ROADMAP", right_layer="HYBRID") ```

m.split_map(.....   Is not working .  

2.  Once split is working how do we get planet imaginary, would like to see the changes to my points between months.

**Comment by giswqs** [comment, marked as answer]
1. Split map can only show tile layer. It will always stay on top of other layers. 
2. Look at the notebook example: https://leafmap.org/notebooks/21_ts_inspector. You will need a Planet API key to see the imagery.

**Reply by Niko-La** [reply]
@giswqs 

1. does that mean point / pins can not be displayed when using split-map?  I have poi like  the image below with pop up-data.  Trying to get split for the image below with planet imagery
![image](https://user-images.githubusercontent.com/14004559/122705047-ec4b2200-d222-11eb-8628-832e1306c39a.png)

2. Got the 21 ts_inspector working on with split-map planet imagery.  Looked at the geemap documentation for inspector but I don't see a way to select coordinate and zoom  
ie  
``` 
m = leafmap.Map(center=[50, 19], zoom=17)
m 
````
Is it possible to have inspector do the split on a specific zoom and coordinate?
